repository/c.json

```diff
@@ -2067,6 +2067,17 @@
 			]
 		},
 		{
+			"name": "Color Schemes - Wicked Schemes by b0xw00d",
+			"details": "https://github.com/b0xw00d/wicked_color_schemes",
+			"labels": ["theme", "color scheme", "syntax highlighting", "wicked"],
```

Comment:
I don't think `syntax highlighting` is correct here, unless I missed a `.tmLanguage` or `.sublime-syntax` file in the repo.

Additionally, `theme` is for Sublime Text themes that skin the whole UI.